src/framegrab/cli/preview.py

```diff
@@ -44,15 +44,16 @@ def get_image_sources_from_config(config: str) -> list:
 
 
 @click.command()
-# @click.argument('config', type=click.Path(exists=True), help="Path to the configuration file.")
 @click.argument("config", type=click.Path(exists=True))
 @click.argument(
     "output",
     type=click.Choice(["imgcat", "cv2", "ascii"], case_sensitive=False),
     default="imgcat",
 )
 def preview(config: str, output: str):
```

Comment:
In the future, it might be cool to also have the ability to preview the results of an autodiscovery

Reply:
Good suggestion!  `framegrab auto-discover --preview` maybe.